kubernetes-tests/src/test/java/io/fabric8/kubernetes/client/mock/SubjectAccessReviewAuthTest.java

```diff
@@ -44,19 +48,100 @@
   private static final Logger logger = LoggerFactory.getLogger(ClusterRoleBindingCrudTest.class);
 
   @Test
-  public void createSubjectAccessReviewTest() {
+  public void testListRulesReview() {
+    KubernetesClient client = server.getClient();
+
+    createSubjectAccessReviewTest();
+
+    SelfSubjectRulesReview selfSubjectRulesReview = new SelfSubjectRulesReviewBuilder()
+            .withNewSpec()
+            .withNamespace("test")
+            .endSpec()
+            .withNewStatus()
+            .withResourceRules(new ResourceRuleBuilder()
+                    .withResourceNames("pod")
+                    .withVerbs("list").build()).endStatus().build();
+
+    SelfSubjectRulesReview rulesReviewResponse = client.subjectAccessReviewAuth().list().create(selfSubjectRulesReview);
+
+    assertNotNull(rulesReviewResponse);
+  }
+
+  @Test
+  public void testLoadFromFileRulesReview() {
+    KubernetesClient client = server.getClient();
+
+    createSubjectAccessReviewTest();
+
+    ObjectMapper mapper = new ObjectMapper(new YAMLFactory());
+
+    SelfSubjectRulesReview review = null;
+
+    try {
+      review = mapper.readValue(getClass().getResourceAsStream("/test-selfsubjectrulesreview.yml"), SelfSubjectRulesReview.class);
+    } catch (IOException e) {
+      e.printStackTrace();
+    }
+
+    assertNotNull(review);
+
+    SelfSubjectRulesReview rulesReviewResponse = client.subjectAccessReviewAuth().list().create(review);
+
+    assertNotNull(rulesReviewResponse);
+  }
+
+  @Test
+  public void testGetRulesReview() {
+    KubernetesClient client = server.getClient();
+
+    createSubjectAccessReviewTest();
+
+    SelfSubjectRulesReview selfSubjectRulesReview = new SelfSubjectRulesReviewBuilder()
+            .withNewSpec()
+            .withNamespace("test")
+            .endSpec()
+            .withNewStatus()
+            .withResourceRules(new ResourceRuleBuilder()
+                    .withResourceNames("pod")
+                    .withVerbs("get").build()).endStatus().build();
```

Comment:
Remove duplicate code, you can write these tests in the same function.
Maybe something like createSelfSubjectRulesReviewTest()
Don't delete the existing test createSubjectAccessReviewTest()